### ISSUE TITLE
🐛 Bug Report: Development and Build errors trying to run website locally

### ISSUE THREAD

**Issue by ChiragAgg5k** [appwrite/website#1499]
### 👟 Reproduction steps

1. Clone the repo.
2. As instructed, run `pnpm install`
3. run `pnpm dev`

### 👍 Expected behavior

Should run the development server with no issues

### 👎 Actual Behavior

Leads to a lot of errors, probably because of undefined environment variables:

![Image](https://github.com/user-attachments/assets/d6715d11-7b55-4668-a5e9-9e42c73a745b)
(during dev)

![Image](https://github.com/user-attachments/assets/c2b1e5d2-9192-4ef1-8b17-5b1cfd51a849)
(during build)

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

MacOS

### 👀 Have you spent some time to check if this issue has been raised before?

- [x] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [x] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by ItzNotABug** [comment]
Look for the `.env.example` file in the root of the project, rename it to `.env`. Restart the dev server and it should work fine.

**Comment by ChiragAgg5k** [comment]
yeah i get that, but i just expected the docs in `CONTRIBUTING.md` to mention this. currently its not mentioned
